name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -403,13 +402,14 @@ protected boolean isIgnoreContentType()
     {
         try
         {
-            String escapedUrl = new URI(url).toASCIIString();
-            return parse(escapedUrl, Jsoup.connect(escapedUrl).userAgent(getUserAgent())
-                            .ignoreContentType(isIgnoreContentType()).timeout(30000).get(), errors);
+            URL urlObject = new URL(url);
+            String html = new WebAccess(urlObject.getHost(), urlObject.getPath()).withScheme(urlObject.getProtocol())
```

Comment:
Ich habe es gerade mal probiert und es erscheint mir als würden die Parameter nicht an WebAccess weitergereicht. Darum kann ich in meiner Datei auch keine Kurse laden weil die ISIN (oder ähnliche Werte) meist als Parameter weitergegeben werden.

Vielleicht braucht es noch einen weiteren Constructor ```new WebAccess(URL url)``` womit man die schon existierende URL übergeben kann (im Gegensatz zu dem URIBuilder in der WebAccess Klasse). Und dann müssten wir uns überlegen wenn anschließend noch ein ```#addParameter``` aufgerufen wird: Entweder wird ein Fehler geworfen oder der Aufruf wird um den Parameter erweitert.

Reply:
Ich klammere den HTMLTableQuoteFeed.java & CS Download von der generellen Klasse aus, da in der CS Klasse der Content einer Excel-Datei an JSON weitergereicht wird.

Eine generelle Änderung auf JSON ist auch nicht wirklich möglich, da bspw. bei Yahoo der Sicherheitstoken nicht übertragen werden kann.